test/evergreen.yml

```diff
@@ -5418,6 +5418,16 @@ buildvariants:
   tasks:
```

Comment:
Interestingly enough the above `! Ubuntu 20.04 Compilers` task runs on `ubuntu2004-wt-build`. I don't know if theres an ARM equivalent.